Trappist/src/Promact.Trappist.Core/Controllers/TestController.cs

```diff
@@ -120,11 +120,18 @@ public TestsController(ITestsRepository testRepository, IStringConstants stringC
             return Ok(testObject);
         }
         [HttpGet("isPausedResume/{id}/{isPause}")]
-        public async Task<IActionResult> PauseTest([FromRoute] int id,[FromRoute] bool isPause)
+        public async Task<IActionResult> PauseTestAsync([FromRoute] int id,[FromRoute] bool isPause)
         {
-            await _testRepository.PauseResumeTest(id,isPause);
+            await _testRepository.PauseResumeTestAsync(id,isPause);
             return Ok(isPause);
         }
+
+        [HttpDelete("deleteIp/{id}")]
+        public async Task<IActionResult> DeleteIPAsync([FromRoute] int id)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename method 'DeleteIPAsync' to match camel case naming rules, consider using 'DeleteIpAsync'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS100)